tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -11,61 +13,122 @@ const keyPrefix = `${uuid()}-key-prefix`;
 const key = `${keyPrefix}/object`;
 
 const cloudServer = new LifecycleUtility(scalityS3Client)
-    .setBucket(srcBucket)
-    .setKeyPrefix(keyPrefix)
-    .setKey(key)
-    .setDestinationLocation(process.env.AWS_BACKEND_DESTINATION_LOCATION);
+      .setBucket(srcBucket)
+      .setKeyPrefix(keyPrefix)
+      .setKey(key);
 
-const s3 = new LifecycleUtility(awsS3Client)
-    .setBucket(process.env.AWS_CRR_BUCKET_NAME)
-    .setKeyPrefix(keyPrefix)
-    .setKey(`${srcBucket}/${key}`);
+const cloud = new LifecycleUtility(awsS3Client, sharedBlobSvc, gcpStorage)
+      .setKeyPrefix(keyPrefix)
+      .setKey(`${srcBucket}/${key}`);
 
 function compareTransitionedData(cb) {
     return series([
         next => cloudServer.getObject(next),
         next => cloudServer.putBucketLifecycleConfiguration(new Date(), next),
         next => cloudServer.waitUntilTransitioned(next),
-        next => s3.getObject(next),
+        next => cloud.getObjectDataFromLocation(next),
+        next => cloudServer.getObject(next),
     ], (err, data) => {
         if (err) {
             return cb(err);
         }
-        cloudServer._compareObjectBody(data[0].Body, data[3].Body);
+        // check data stored on the target cloud
+        cloudServer._compareObjectBody(data[0].Body, data[3]);
+        // check that object is still readable from Zenko
+        cloudServer._compareObjectBody(data[0].Body, data[4].Body);
         return cb();
     });
 };
 
-describe('transition to AWS backend', function() {
-    this.timeout(LIFECYCLE_TIMEOUT);
+const locationParams = {
+    local: {
```

Comment:
Minor: since we're using the names of the public clouds, I think it reads a bit easier if this is 'Zenko'.